css/indexStyle.css

```
@@ -0,0 +1,384 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  /* 폰트 색상 weight 변수 */
+  --primary:#6d6afd;
+  --red:#FF5B56;
+  --black:#111322;
+  --gray-300:#3E3E43;
+  --gray-200:#9FA6B2;
+  --gray-100:#CCD5E3;
+  --lightGray-100:#E7EFFB;
+  --lightGray-200:#F0F6FF;
+
+
+  --logo-weight:700;
+  margin: 0;
+  font-family: "Montserrat Alternates", 'Noto Sans KR', sans-serif;
+}
+
+.header-container {
+  background-color: var(--lightGray-200);
+}
+
+
+/* 인덱스 페이지 a태그는 전부 decoration none을 사용해서 바꾸지 않아도 될거같습니다. */
+a {
+  text-decoration: none;
+}
```

@wjy308 

```suggestion
/* 인덱스 페이지 a태그는 전부 decoration none을 사용해서 바꾸지 않아도 될거같습니다. */

```
랜딩페이지 뿐만 아니라 보통 ㅐㄷ부분의 페이지에서 링크가 기본적으로 제공하는 underline 스타일을 활용하지 않는 경우가 많은데요
이처럼 무든 페이지에서 공통적으로 활용될 수 있는 스타일들은 `global.css` 또는 `common.css` 에 모아서 관리해보면 어떨까요?